config/deploy/dev.rb

```diff
@@ -0,0 +1 @@
+server 'upaya-dev.18f.gov', roles: %w(web app db)
```

Comment:
This is missing `user: 'ubuntu'` as one of the parameters. Thanks to @amoose for pointing me here.